### ISSUE TITLE
Add support for .mks & .mka files

### ISSUE THREAD

**Issue by op200** [clsid2/mpc-hc#3121]
Similar to #1070 

### About the importance of mks / mka
* The subtitle filter is not compatible with the fonts included in the ass, but it is compatible with the fonts read by the player.
* We can read chapter info from mks.
* We can read plural audio or subtitles from mka.
* When there are multiple subtitles, mks can make the subset-fonts smaller.
* The existence of mks / mka can keep the original video independent and improve file reusability.
* As a universal and enduring subtitle format, mks deserves to be compatible with all players.

To sum up, although mks / mka is unpopular in many fields, it is still worth using.   
Using mks / mka will bring great convenience to users.

**Comment by NekomiAyase** [comment]
+1
